lib/tasks/ezid/ezid_logger.rb

```diff
@@ -0,0 +1,33 @@
+#Custom Migration Logger
+require 'singleton'
+
+class EzidLogger < Logger
```

Comment:
I think it was used in the old rake task that I removed. This is probably obsolete as well. Will remove the file.